src/field/input_features.md

```diff
@@ -39,15 +38,42 @@ Once you are finished, press **Done**. If your feature is an area, the shape wil
 
 ![Capturing lines](./input-capture-line.png)
 
-## Edit or delete features
+### Snapping features
+<Badge text="since Input 1.6.0" type="tip"/>
+
+Snapping can be enabled in your <MainPlatformName /> project to make the field survey easier. [How to Set Up Snapping](../gis/snapping/) will guide you through the snapping options.
+
+If snapping is enabled, the cross hairs will turn purple and snap to vertices (left) or segments (right) of existing features when capturing new features.
+![input snapping](../gis/snapping/input_basic_snapping.png)
+
+## Edit features
 Tap on a feature and press the edit button.
-![Edit button](../tutorials/mobile/merginmaps-mobile-edit-button.jpg)
+![Edit button](./input-edit.png)
 
 Now you can change the attributes in the form and modify the shape of the feature using **Edit geometry**. Adjust the location by panning the map or use **GPS** in the same manner as when adding new features. Once you are happy with your changes, press **Save**.
+![Edit geometry](./input-edit-geometry.png)
+
+### Split geometry of lines or areas
+<Badge text="since Input 1.6.0" type="tip"/>
+Lines and areas can be split into two parts. Both parts will keep the same attributes. One of them will get a new `fid`.
+
+To split geometry of a feature:
+1. Tap on a feature and press the edit button.
+![Edit button](./input-edit-feature.png)
+
+2. Tap **Split geometry**
+![split geometry](./input-split-geometry.png)
+
+3. Create the splitting line by adding points. When finished, tap **Done**
+![split geometry](./input-split-geom-point.png)
+
+4. Two individual features are created!
```

Comment:
Well it can actually be more than two features, it depends on how many times the line crosses the geometry 🙂